SourceX/dx.cpp

```diff
@@ -223,6 +230,12 @@ void LimitFrameRate()
 void RenderPresent()
 {
 	SDL_Surface *surface = GetOutputSurface();
+#ifdef __AMIGA__
+	if (ac68080)	{
+		old_buffer = surface->pixels;
+		surface->pixels = (void*)(~31 & (31+(Uint32)old_buffer));

```

Comment:
this has been added to the Amiga SDL, and is a bit moot with Sam's ASM PR, so it should probably be removed

Reply:
@arczi84 Just to double check, this stuff was added in our SDL, right?

Reply:
**Resolved**